### ISSUE TITLE
Request for the material file of Iodine

### ISSUE THREAD

**Issue by z0gSh1u** [DIDSR/MCGPU#2]
Hello! MCGPU is a very easy-to-use tool for CT imaging researches. Thanks for your beautiful works.

I'm wondering if we can have the material file of Iodine, since it is very useful in Dual-Energy CT researches and Contrast Imaging.

I've checked all existing material sources, and none of them includes the Iodine.
- https://github.com/DIDSR/VICTRE_MCGPU/tree/master/example_simulations/material
- https://github.com/DIDSR/MCGPU-PET/tree/main/sample_simulation/materials
- The `common_materials_PENELOPE2006` archive

I've also tried to generate it myself using the Fortran program `MC-GPU_create_material_data.f`. However, the oldest PENELOPE I can access to is version 2011, while the file format for Rayleigh scattering changed since version 2008 (see https://code.google.com/archive/p/mcgpu/issues/7), so I failed to create material data for Iodine.

**Comment by z0gSh1u** [comment]
For those who need PENELOPE material executable, please refer to the [pyPENELOPE](https://sourceforge.net/projects/pypenelope/files/). Older version integrates older PENELOPE from 2008 to 2011.

**Comment by andreubs** [comment]
Hi,

Happy to hear that the code is working for your research. 

Contrast agents are tricky to model because after injection they are mixture of iodine, blood and other components. There is clearly not 100% (solid) iodine injected into the patient.

I have uploaded a material file I have used in the past to model contrast agent:
https://github.com/DIDSR/MCGPU/blob/master/materials/blood90_iodine10__5-120keV.mcgpu.gz

This material file for MCGPU, based on PENELOPE 2006, contains 90% blood and 10% iodine (the blood composition is taken from Woodard et al, Bri. Jour. Rad. 59 (1986)).

Best regards,

    Andreu
